packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -100,6 +103,12 @@ export class Stepper {
   connectedCallback(): void {
     this.mutationObserver?.observe(this.el, { childList: true });
     this.updateItems();
+    this.resizeObserver?.observe(document.body);
```

Comment:
Question: Why are we observing the document body? Shouldn't we just be observing a component's size?

Reply:
The component do not collapse entirely when viewport size decreases as per spec to avoid shrinking stepper-items. To identify if the viewport size has decreased and showcase UX for small screen when required we are observing `document.body`.  

Update:
After discussing with Skye regarding this, the suggestion is to have a min-width on each stepper item which helps us to calculate the breakpoint width more precisely. Changes coming soon 🚧